pages/agent/v3/gcloud.md

```diff
@@ -2,6 +2,12 @@
 
 The Buildkite Agent can be run on [Google Cloud Platform](https://cloud.google.com). For fine control over long–lived agents, you might like to run the agent using individual VM instances on Google Compute Engine. Or run Docker–based builds using a scalable cluster of agents on the Google Kubernetes Engine using Kubernetes.
 
+## Using the Buildkite Agent Stack for Kubernetes on the Google Cloud Platform
+
+The Buildkite Agent's jobs can be run within a Kubernetes cluster on Google Cloud Platform.
+
+Learn more about how to set up the Buildkite Agent stack to run in Kubernetes in [Buildkite Agent Stack for Kubernetes](https://github.com/buildkite/agent-stack-k8s?tab=readme-ov-file#buildkite-agent-stack-for-kubernetes).
```

Comment:
```suggestion
Once your Kubernetes cluster is running on GCP, follow the [Buildkite Agent Stack for Kubernetes](https://github.com/buildkite/agent-stack-k8s?tab=readme-ov-file#buildkite-agent-stack-for-kubernetes) instructions to set up the Buildkite Agent stack to run in Kubernetes.
```

Reply:
this is minor, but just thinking we can go with nearly identical language between AWS and GCP if we want to

Reply:
Thanks for clarifying this!